### PR TITLE
Web3War Token Listing

### PR DESCRIPTION
**Pull request by WuBruno**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Web3War

##### Twitter Link:
https://twitter.com/web3war_game

##### List of audit links if any:
[?]

##### Website Link:
https://www.w3w.game/

##### Logo (High resolution, will be shown with rounded borders):
FPSToken

##### Treasury Addresses (if the protocol has treasury)
[?]

##### Chain:
Zilliqa & BSC 

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
web3war

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
29092

##### Short Description (to be shown on DefiLlama):
Web3War® (W3W) is the first-person shooter crafted by the industry-leading game developer Roll1ng Thund3rz and incubated by Zilliqa, the trailblazer in blockchain technology and the OG of Layer 1.

W3W masterfully fuses time-honored gaming principles with innovative Skill-to-Earn (‘S2E’) mechanics. The project promises to deliver as the first ‘Web3 game for all’ — a multiplayer, free-to-play, seasonal content-oriented, first-person shooter (FPS) that prioritizes fun and high-quality gameplay above all else.

Already earning rave reviews from players and captivating gaming heavyweights and ‘Big Tech’, W3W has been endorsed by several elite esports teams, including Mad Lions, RRQ Esports, and Ninjas in Pyjamas. The game has established close collaboration with tech giants Google and Microsoft, becoming one of the first blockchain games available on the Microsoft Store.

##### Token address and ticker if any:
BSC: FPS https://bscscan.com/address/0x351da1e7500aba1d168b9435dce73415718d212f#code
Zilliqa: FPS https://viewblock.io/zilliqa/address/zil1j2wrzjljwyjelspmtr63vfl34c467ype2w3mjl (or equivalently https://otterscan.zilliqa.com/address/0x241c677D9969419800402521ae87C411897A029f)

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Gaming

##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:
No oracles used

##### forkedFrom (Does your project originate from another project):
Not forked

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL is the total valuation of the token across the chains the token exists

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/Zilliqa

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/web3war exports TVL: 
        
 
 
    zilliqa                   0
    bsc                       3.10 M
    
    total                    3.10 M

**Comment by llamatester** [comment]
The adapter at projects/web3war exports TVL: 
        
 
 
    zilliqa                   0
    bsc                       3.09 M
    
    total                    3.09 M

**Comment by llamatester** [comment]
The adapter at projects/web3war exports TVL: 
        
 
 
    zilliqa                   0
    bsc                       3.07 M
    
    total                    3.07 M

**Comment by llamatester** [comment]
The adapter at projects/web3war exports TVL: 
        
 
 
    zilliqa                   0
    bsc                       3.07 M
    
    total                    3.07 M

File: projects/web3war/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    console.log(supply);
+
+    api.add(BSC_FPS_TOKEN_CONTRACT, supply);
+  }
+}
+
+module.exports = {
+  timetravel: true,
+  misrepresentedTokens: false,
+  methodology: "counts number of FPS tokens across all chains",
```
Comment (g1nt0ki) [inline projects/web3war/index.js:37]: we cant count your token supply as tvl: https://docs.llama.fi/list-your-project/what-to-include-as-tvl

**Pull request closed without merging**